homeassistant/components/august/__init__.py

```diff
@@ -180,7 +187,10 @@ def __init__(self, hass, api, access_token):
         self._access_token = access_token
         self._doorbells = self._api.get_doorbells(self._access_token) or []
         self._locks = self._api.get_operable_locks(self._access_token) or []
-        self._house_ids = [d.house_id for d in self._doorbells + self._locks]
+        self._house_ids = []
+        for device in self._doorbells + self._locks:
+            if device.house_id not in self._house_ids:
```

Comment:
If we want this container to have unique items we can use a set instead of a list. Then we can simplify here since a set by default won't add duplicate items.

Reply:
will do